server/pytest.ini

```diff
@@ -1,3 +1,15 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = portal.settings.unit_test_settings
 python_files = tests.py *unit_test.py
+
+filterwarnings =
+    ignore::DeprecationWarning:django.*:
+    ignore::DeprecationWarning:openapi_schema_validator.*:
+    ignore::DeprecationWarning:openapi_spec_validator.*:
+    ignore::DeprecationWarning:openapi_core.*:
+    ignore::DeprecationWarning:kombu.*:
+    ignore::DeprecationWarning:pkg_resources.*:
+    ignore:.*Django now detects this configuration.*:django.utils.deprecation.RemovedInDjango41Warning
```

Comment:
Do we want these warnings removed? I think it might be better if we resolved the deprecation warnings, as in changed code to resolve the warnings, rather than suppressed the warnings.

Reply:
@van-go , just so that we do not ignore this permanently, would it be possible to add tracking jira tasks and add the task number to the ini file (comments start with ';' in ini file, afaik).
+ django version fix: [4.2.4](https://github.com/django/django/releases/tag/4.2.4)
+ openapi: <I do not which version they fixed the warning>
+ kombu: 
+ which third party throws : django.utils.deprecation.RemovedInDjango41Warning 
...

Adding a note in task to remove the ignore when the task is fixed will ensure that the ignore is not left behind permanently.